### DISCUSSION TITLE
Infected routers

### DISCUSSION THREAD

**Discussion by 737simpilot** [CIDRAM/CIDRAM#306] [General]
There are ports that are open to the wold on some people's routers or modems. These ports will often show up at places like Shodan, Census, alienvault, etc. The biggest port on some routers or modems to be attacked is 7547. You can read about that [here](https://isc.sans.edu/forums/diary/Port+7547+SOAP+Remote+Code+Execution+Attack+Against+DSL+Modems/21759/). Here's one file and its results at [VirusTotal](https://www.virustotal.com/gui/file/7e84a8a74e93e567a6e7f781ab5764fe3bbc12c868b89e5c5c79924d5d5742e2/detection) You'll note the lame antivirus engines that never picked up on this one file. The other hashes provided are similar to detection results from this one. 

If you see an IP address that is suspect of being a spammer and whatnot, check it for this or other open ports. It doesn't necessarily mean the router or modem belonging to that IP is infected and thus serving as a launch pad for nefarious activity, but it's a start on investigations.  If of course you see web ports open on the IP then you can reasonably assume it's of nefarious purposes. But again, you have to be careful on interpreting your results. I've seen many IPs particularly out of Australia have an open port for the SIP protocol. This is used for VoIP so would most likely be benign. On the other hand, since it is an open port to the world, a hacker could infect the VoIP server or hardware for certain uses... If you run something like this, it would be in your best interest to explore the hardware based firewall pfSense.  You can take pfSense for a test drive in VMware Workstation player or maybe Virtual Box. I prefer VMware. 

Now, having said all this. Today I noticed in my logs and investigation of past logs that a user has been trying shenanigans. Mostly with creating more than one account or trying to login to my account. Though, I'm not proof positive it's the same person, but the IPs are not only of the same CIDR but ASN. The IPs belong to a popular residential ISP here in the U.S. called Frontier. The ASN for these IPs fall within AS5650. Upon my research I discovered open ports on a lot of Frontier's IPs in this ASN for ports 4567 and 3128. Worse yet, some of these routers or modems show up as infected at alienvault.  [here's](https://otx.alienvault.com/indicator/ip/23.239.191.105) just one example.  I literally just randomly entered IPs within some of the CIDRs in this ASN at alienvault and here's a list of some IPs that currently have infections.

```
47.187.10.100

47.183.11.103

47.183.110.96

47.183.20.70

23.239.191.20

23.239.191.105
```

Also, this ASN is multiplay. They have some CIDRs with the lovely description of, ["Server Destroyers LLC ."](https://opencorporates.com/companies/us_wy/2018-000814122)

And doing research for port 4567, I see [Verizon](https://forums.verizon.com/t5/fios-internet/guy-accessed-remote-administration-port-4567-on-my-router-thanks/td-p/241017) does the same crap with opening a port to the world for administrative purposes. To those that don't know. Verizon is a U.S. company who primarily started out as a cell carrier and moved into the ISP market and God only knows what else. They offer Verizon FIOS (fiber Internet) in the most populace cities in the U.S. like in states of New York, New Jersey and I think in Texas and other places. 

My town just installed their own fiber ISP and I'll probably take advantage of that. 1 Gig up and down, yo! LOL! It was created by vote a couple years back. The vote succeeded and now they rolled it out to most residents in town thus far and are continually doing so. The town north of here already had their town install fiber. 

In conclusion, if the IP address has an open port, it's like an RSVP to come on in to the whole world and thus _could_ be used as a launching pad for nefarious activity.
